### ISSUE TITLE
Incorrect peer dependency

### ISSUE THREAD

**Issue by hasusozam** [apollographql/apollo-server#1656]
> [4/5] Linking dependencies...
warning "apollo-server > graphql-subscriptions@0.5.8" has incorrect peer dependency "graphql@^0.10.5 || ^0.11.3 || ^0.12.0 || ^0.13.0".
warning "apollo-server > apollo-server-core > @apollographql/apollo-upload-server@5.0.3" has incorrect peer dependency "graphql@^0.13.1".
warning "apollo-server > graphql-tools@3.1.1" has incorrect peer dependency "graphql@^0.13.0".
warning "apollo-server > apollo-server-core > graphql-tag@2.9.2" has incorrect peer dependency "graphql@^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 ||
^0.13.0".
warning "apollo-server > apollo-server-core > subscriptions-transport-ws@0.9.14" has incorrect peer dependency "graphql@^0.10.0 || ^0.11.0 || ^0.12
.0 || ^0.13.1".
warning "apollo-server > graphql-tools > apollo-link@1.2.2" has incorrect peer dependency "graphql@^0.11.3 || ^0.12.3 || ^0.13.0".

Here is my `package.json`:

```
  "dependencies": {
    "apollo-server": "^2.0.7",
    "dotenv": "^6.0.0",
    "esm": "^3.0.82",
    "graphql": "^14.0.2"
  },
```

In order to get the same result you just have to: `yarn add apollo-server graphql`

**Comment by ksmithut** [comment]
I think this is because graphql 14 is new and apollo-server hasn't been updated. If you change your graphql version to be "^0.13.2" it will be fixed until they update their peer dependency. Their docs should be updated to pin graphql to that version until they do, though.

**Comment by hasusozam** [comment]
Thank you for pointing it out. I did and the warnings went away. Thank you.

**Comment by zefexdeveloper** [comment]
![image](https://user-images.githubusercontent.com/30276092/47308980-96ed8b80-d609-11e8-8158-59ce577dcc6a.png)

Is it safe to ignore or I really have to use use 0.13?

**Comment by cheapsteak** [comment]
Hi folks!
Thanks for the input and apologies for the inconvenience

The peer dependencies for both [graphql-subscriptions](https://github.com/apollographql/graphql-subscriptions/blob/fad56889ab13c8d2a39b5db9592981c517e778b5/package.json#L13-L14) and [graphql-tools](https://github.com/apollographql/graphql-tools/blob/67d03a94667bd5fba2e93d286cd7918d937a3907/package.json#L58-L59) has since been updated to allow graphql@14

We've also switched from `apollo-upload-server` to `graphql-upload`, which has also updated its [peer dependency](https://github.com/jaydenseric/graphql-upload/blob/d9d1972930442dd24b70f976ffa6fde566faaf75/package.json#L35-L36) to support graphql@14

I believe this issue should now be safe to close? Please let us know if this regresses 🙏

**Comment by sudipta17** [comment]
Mac OS 10.15.7
tried installing **npm install -g apollo** in terminal
It still gives me the error
![Screenshot 2020-10-01 at 11 41 29 AM](https://user-images.githubusercontent.com/33668301/94774229-0e71eb00-03db-11eb-9c4e-d5d7ef7d06db.png)

**Comment by alexburner** [comment]
Looks like this is happening again, for `graphql@15`

**Comment by glasser** [comment]
@sudipta17 The `apollo` package isn't part of this repository. It's part of the https://github.com/apollographql/apollo-tooling repo. (And full disclosure — the Apollo CLI is not under active development; we're working to break it into a few parts served by more focused tools.)

@alexburner I'm sorry you're seeing this issue but your report doesn't have enough information for me to help. What did you do and what did you see?

**Comment by alexburner** [comment]
@glasser sorry for the late response. I was seeing the same `incorrect peer dependency` errors as the original issue, but with `graphql@15` instead of 14.

However, I'm not seeing it now. My npm package.json versions are exact... so perhaps upgrading to npm 7 fixed it?

**Comment by glasser** [comment]
@alexburner Probably fixed by Apollo Server v2.21.
